src/main/java/com/labsynch/labseer/api/ApiExperimentController.java

```diff
@@ -1031,22 +1031,20 @@ public ResponseEntity<java.lang.String> updateFromJsonArray(@RequestBody String
 	// return new ResponseEntity<String>(headers, HttpStatus.OK);
 	// }
 
-	// @RequestMapping(value = "/seldelete/{id}", method = RequestMethod.DELETE,
-	// headers = "Accept=application/json")
-	// public ResponseEntity<String> trueDeleteById(@PathVariable("id") Long id) {
-	// Experiment experiment = Experiment.findExperiment(id);
-	// HttpHeaders headers = new HttpHeaders();
-	// headers.add("Content-Type", "application/json; charset=utf-8");
-	// if (experiment == null) {
-	// return new ResponseEntity<String>(headers, HttpStatus.NOT_FOUND);
-	// }
-	// long startTime = new Date().getTime();
-	// Experiment.removeExperimentFullCascade(id);
-	// long endTime = new Date().getTime();
-	// long totalTime = endTime - startTime;
-	// logger.info(" total elapsed time: " + totalTime + " ms");
-	// return new ResponseEntity<String>(headers, HttpStatus.OK);
-	// }
+	@RequestMapping(value = "/seldelete/{id}", method = RequestMethod.DELETE, headers = "Accept=application/json")
+	public ResponseEntity<String> selDeleteById(@PathVariable("id") Long id) {
+		Experiment experiment = Experiment.findExperiment(id);
+		HttpHeaders headers = new HttpHeaders();
+		headers.add("Content-Type", "application/json; charset=utf-8");
```

Comment:
the OO way would be `header.setContentType(MediaType.APPLICATION_JSON)` (or `MediaType.APPLICATION_JSON_UTF8` if you need to support old browsers).

Reply:
Good to know. We have strings like this all over the code, so @dalejerikson no need to fix it on this PR, but we can keep an eye out for that in the future.